src/Symfony/Component/HttpClient/composer.json

```diff
@@ -21,7 +21,7 @@
     "require": {
         "php": "^7.1.3",
         "psr/log": "^1.0",
-        "symfony/http-client-contracts": "^1.1",
+        "symfony/http-client-contracts": "^1.1.3",
```

Comment:
There is no 1.1.2 release yet (only for the `symfony/contracts` package). Is that intended?

Reply:
Yes, that's how we have to do versioning: we must match the versions of contracts itself.